README.md

```diff
@@ -2,13 +2,21 @@
 
 Easily setup and use backbone.js (1.2.2) with Rails 3.1 and greater
 
-##Version##
+## Install Notes for Galvanize
+
+For some reason, I couldn't get the gems to install on the latest bundler, so use the command:
+
+> bundle _2.1.4_ install
```

Comment:
Is this going to be a problem when we merge this into strategy?

Reply:
Not at all, all tests pass when strategy uses this branch: https://github.com/acl-services/aclrisk/pull/3353
Though I'm going to update that PR to use the main branch once this is merged 😄